src/game/client/ingameui.cpp

```diff
@@ -13,35 +13,1131 @@
+                                g_theGlobalLanguage->Adjust_Font_Size(m_namedTimerNormalPointSize),
+                                m_namedTimerNormalBold));
+                        }
+                    } else {
+                        if (timer_info->m_timestamp == 0 || timer_info->m_countdown) {
+                            timer_info->m_displayString->Set_Font(g_theFontLibrary->Get_Font(m_namedTimerReadyFont,
+                                g_theGlobalLanguage->Adjust_Font_Size(m_namedTimerReadyPointSize),
+                                m_namedTimerReadyBold));
+                        }
+                    }
+
+                    timer_info->m_timestamp = timer_frame2;
+                    int minutes = timer_frame2 / 60;
+                    int seconds = timer_frame2 % 60;
+
+                    if (timer_info->m_countdown) {
+                        if (seconds < 10) {
+                            timer_time.Format(U_CHAR("%s %d:0%d"), timer_info->m_timerText.Str(), minutes, seconds);
+                        } else {
+                            timer_time.Format(U_CHAR("%s %d:%d"), timer_info->m_timerText.Str(), minutes, seconds);
+                        }
+                    } else {
+                        timer_time.Format(U_CHAR("%s %d"), timer_info->m_timerText.Str(), timer_value);
+                    }
+
+                    timer_info->m_displayString->Set_Text(timer_time);
+                }
+
+                int timer_x_aligned = timer_x;
+
+                if (is_right_aligned) {
+                    timer_x_aligned -= timer_info->m_displayString->Get_Width(-1);
+                }
+
+                if (timer_frame2 != 0 || !timer_info->m_countdown || this->m_namedTimerFlashDuration == 0.0f) {
+                    timer_info->m_displayString->Draw(timer_x_aligned, timer_y, timer_info->m_color, border_color);
+                } else {
+                    if (g_theGameLogic->Get_Frame()
+                        >= (unsigned int)m_namedTimerFlashDuration + m_namedTimerLastFlashFrame) {
+                        m_namedTimerUsedFlashColor = !m_namedTimerUsedFlashColor;
+                        m_namedTimerLastFlashFrame = g_theGameLogic->Get_Frame();
+                    }
+
+                    timer_info->m_displayString->Draw(timer_x_aligned,
+                        timer_y,
+                        m_namedTimerUsedFlashColor ? timer_info->m_color : m_namedTimerFlashColor,
+                        border_color);
+                }
+
+                timer_y -= timer_info->m_displayString->Get_Font()->m_height;
+            }
+        }
+    }
+
+    if (g_theLookAtTranslator != nullptr) {
+        if (m_drawRMBScrollAnchor) {
+            const ICoord2D *anchor = g_theLookAtTranslator->Get_RMB_Scroll_Anchor();
+
+            if (anchor != nullptr) {
+                static int mainColor = Make_Color(0, 255, 0, 255);
+                static int dropColor = Make_Color(0, 0, 0, 255);

```

Comment:
This can be constexpr because `Make_Color` is constexpr and all inputs are constants.